libs/core-ui/src/lib/util/StatisticsUtils.ts

```diff
@@ -277,6 +278,9 @@ export const generateMetrics: (
       return generateImageStats(trueYSubset, predYSubset);
     });
   }
+  if (modelType == ModelTypes.ObjectDetection) {
```

Comment:
Looks like TS uses === not == for equals

Reply:
Omg never even noticed, thanks a lot!